ratis-assembly/src/main/assembly/bin.xml

```diff
@@ -37,6 +37,7 @@
         <include>org.apache.ratis:ratis-logservice</include>
         <include>org.apache.ratis:ratis-netty</include>
         <include>org.apache.ratis:ratis-proto</include>
+        <include>org.apache.ratis:ratis-docs</include>
```

Comment:
This adds ratis-docs jar to the binary tar.  I think including docs as regular files would provide better user experience.  This can be achieved by an additional `fileSet` below:

```diff
--- ratis-assembly/src/main/assembly/bin.xml
+++ ratis-assembly/src/main/assembly/bin.xml
@@ -37,7 +37,6 @@
         <include>org.apache.ratis:ratis-logservice</include>
         <include>org.apache.ratis:ratis-netty</include>
         <include>org.apache.ratis:ratis-proto</include>
-        <include>org.apache.ratis:ratis-docs</include>
         <include>org.apache.ratis:ratis-replicated-map</include>
         <include>org.apache.ratis:ratis-server</include>
         <include>org.apache.ratis:ratis-test</include>
@@ -78,5 +77,11 @@
       <fileMode>0644</fileMode>
       <directoryMode>0755</directoryMode>
     </fileSet>
+    <fileSet>
+      <directory>${project.basedir}/../ratis-docs/target/classes/docs</directory>
+      <outputDirectory>docs</outputDirectory>
+      <fileMode>0644</fileMode>
+      <directoryMode>0755</directoryMode>
+    </fileSet>
   </fileSets>
 </assembly>
```